js/debugging/demos/tax-calc/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Пример отладки асинхронного кода
+ * */
+
+(async function () {
+  const GROSS_SALARY = 4500
+
+  try {
+    const netSalary = await getNetSalary(GROSS_SALARY)
+
+    console.log("Зарплата после налогов: ", netSalary);
+  } catch (error) {
+    console.warn("Ошибка при вычислении зарплаты после налогов: ", error);
+  }
+}())
+
+/**
+ * @param {number} salaryGross
+ * @returns {Promise<*>}
+ */
+async function getNetSalary (salaryGross) {
+  const taxValueRealResult = await getTaxValueReal('AB')
+
+  if (!taxValueRealResult.error) {
```

Comment:
ДУмаю что лаконичнее будет обойтись без отрицания, как считаешь?
```suggestion
  if (taxValueRealResult.data) {
```

Reply:
Тут надо аккуратно, если  taxValueRealResult.data = 0, то код сломается.